docs/UserGuide.adoc

```diff
@@ -250,3 +306,57 @@ e.g.`select 2`
 * *History* : `history`
 * *Undo* : `undo`
 * *Redo* : `redo`
+
+== Patch Notes
+
+=== Features added in v1.0
+In version v1.0, users will be able to: +
+
+* have their data backed up automatically when Ark is launched.
+
+{{More to be added}}
+
+=== Features added v1.1
+In version v1.1, users will be able to: +
```

Comment:
Perhaps this portion can be updated since we are done with iteration `v1.1` and `v1.2`.